src/services/air.js

```diff
@@ -1,85 +1,148 @@
+const PM_DATA = { text: "PM Level", color: "#757575", textColor: "#fff" };
+
+export const aqiColors = [
+  "#00e400",
+  "#ffff00",
+  "#ff7e00",
+  "#ff0000",
+  "#8f3f97",
+  "#7e0023",
+];
+
+export const aqiRanges = [
+  "0 - 50",
+  "51-100",
+  "101-150",
+  "151-200",
+  "201-300",
+  "301 - 500+",
+];
+
+export const aqiTicks = [0, 50, 100, 150, 200, 300, 500];
+
+export const aqiLegend = [
+  GOOD,
+  MODERATE,
+  UNHEALTHY_SENSITIVE,
+  UNHEALTHY,
+  VERY_UNHEALTHY,
+  HAZARDOUS,
+];
 
 export function getDisplay(type, value) {
-  if (type == 'aqi') {
+  if (type == "aqi") {
     return getAQIDisplay(value);
-  } else if (type == 'pm2_5') {
-    return getPM2_5Display(value);
+    // } else if (type == "pm2_5") {
+    //   return getPM2_5Display(value);
   } else {
-    return getPM10Display(value);
+    return getPM_Display();
   }
 }
 
 // Based on https://www.airnow.gov/aqi/aqi-basics/
 export function getAQIDisplay(value) {
-  switch(true) {
-    case (value === undefined):
+  switch (true) {
+    case value === null:
       return NO_DATA;
-    case (value >= 300):
+    case value >= 300:
       return HAZARDOUS;
-    case (value >= 200):
-      return VERY_UNHEALTHY
-    case (value >= 150):
+    case value >= 200:
+      return VERY_UNHEALTHY;
+    case value >= 150:
       return UNHEALTHY;
-    case (value >= 100):
+    case value >= 100:
       return UNHEALTHY_SENSITIVE;
-    case (value >= 50):
+    case value >= 50:
       return MODERATE;
-    default:
+    case value >= 0:
       return GOOD;
+    default:
+      return NO_DATA;
   }
 }
 
-// Based on https://blissair.com/what-is-pm-2-5.htm
-export function getPM2_5Display(value) {
-  switch(true) {
-    case (value === undefined):
+export function getAQIColor(value) {
+  switch (true) {
+    case value === undefined:
       return NO_DATA;
-    case (value >= 250.5):
-      return HAZARDOUS;
-    case (value >= 150.5):
-      return VERY_UNHEALTHY
-    case (value >= 55.5):
-      return UNHEALTHY;
-    case (value >= 35.5):
-      return UNHEALTHY_SENSITIVE;
-    case (value >= 12.1):
-      return MODERATE;
-    default:
-      return GOOD;
+    case value <= 50:
+      return "#00e400";
+    case value <= 100:
+      return "#ffff00";
+    case value <= 150:
+      return "#ff7e00";
+    case value <= 200:
+      return "#ff0000";
+    case value <= 300:
+      return "#8f3f97";
+    case value <= 301:
+      return "#7e0023";
   }
 }
 
-// Based on https://www.epa.vic.gov.au/for-community/environmental-information/air-quality/pm10-particles-in-the-air
-export function getPM10Display(value) {
-  switch(true) {
-    case (value === undefined):
-      return NO_DATA;
-    case (value > 300):
-      return HAZARDOUS;
-    case (value > 120):
-      return VERY_UNHEALTHY
-    case (value > 80):
-      return UNHEALTHY_SENSITIVE;
-    case (value > 40):
-      return MODERATE;
-    default:
-      return GOOD;
-  }
+// Based on https://blissair.com/what-is-pm-2-5.htm
```

Comment:
I’d just kill all the commented stuff in this file.

Reply:
Done